File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplatesCache/CSharp/Web/1049/AjaxWcfService/Service.svc.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Runtime.Serialization;
using System.ServiceModel;
using System.ServiceModel.Activation;
using System.ServiceModel.Web;
using System.Text;

namespace $rootnamespace$
{
	[ServiceContract(Namespace="")]
	[AspNetCompatibilityRequirements(RequirementsMode=AspNetCompatibilityRequirementsMode.Allowed)]
	public class $safeitemrootname$
	{
		// Чтобы использовать протокол HTTP GET, добавьте атрибут [WebGet]. (По умолчанию ResponseFormat имеет значение WebMessageFormat.Json.)
		// Чтобы создать операцию, возвращающую XML,
		//     добавьте [WebGet(ResponseFormat=WebMessageFormat.Xml)]
		//     и включите следующую строку в текст операции:
		//         WebOperationContext.Current.OutgoingResponse.ContentType = "text/xml";
		[OperationContract]
		public void DoWork()
		{
			// Добавьте здесь реализацию операции
			return;
		}

		// Добавьте здесь дополнительные операции и отметьте их атрибутом [OperationContract]
	}
}

File: 1сем/ОАиП лабы/Common7/IDE/ItemTemplates/VisualBasic/Web/1049/AjaxWcfService/Service.svc.vb
Imports System.ServiceModel
Imports System.ServiceModel.Activation
Imports System.ServiceModel.Web

<ServiceContract(Namespace:="")>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
<AspNetCompatibilityRequirements(RequirementsMode:=AspNetCompatibilityRequirementsMode.Allowed)>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
Public Class $safeitemrootname$

    ' Чтобы использовать протокол HTTP GET, добавьте атрибут <WebGet()>. (По умолчанию ResponseFormat имеет значение WebMessageFormat.Json.)
    ' Чтобы создать операцию, возвращающую XML,
    '     добавьте <WebGet(ResponseFormat:=WebMessageFormat.Xml)>,
    '     и включите следующую строку в текст операции:
    '         WebOperationContext.Current.OutgoingResponse.ContentType = "text/xml"
    <OperationContract()>$if$ ($targetframeworkversion$ <= 3.5) _$endif$
    Public Sub DoWork()
        ' Добавьте здесь реализацию операции
    End Sub

    ' Добавьте здесь дополнительные операции и отметьте их атрибутом <OperationContract()>

End Class
